### PR TITLE
Changes color-scheme meta tag link URL from W3C draft to Whatwg

### PR DESCRIPTION
**Pull request by tylergaw**
**This PR**

_TL;DR; Changes https://drafts.csswg.org/css-color-adjust/#color-scheme-meta to https://html.spec.whatwg.org/multipage/semantics.html#meta-color-scheme_

Changes the URL for link of for more information about the `color-scheme` meta from a hash that no longer exists on the W3C draft page to a working link on the Whatwg spec.

**Why?**

The current URL https://drafts.csswg.org/css-color-adjust/#color-scheme-meta goes to the top of the document because there is no longer an element with the `color-scheme-meta` id in the page.

The only mention of the `color-scheme` meta tag in that draft does not have an `id` to link to it. The note reads:

> Note: [HTML] specifies a color-scheme meta tag which can be used to set the color-scheme on the root element as a non-CSS presentational hint.

In that note there is a link to the `color-scheme` meta tag docs in the Whatwg spec https://html.spec.whatwg.org/multipage/semantics.html#meta-color-scheme

When I was reading the post, I was confused why the link just went to the top of the W3C draft and not to relevant info about the `color-scheme` meta tag.

### CODE REVIEW

**Comment by tylergaw** [comment]
@googlebot I signed it!

**Comment by jpmedley** [comment]
@tomayac I always defer to post authors on PRs like this. Does this change look good to you?

**Review by tomayac — APPROVED** [review]
LGTM, approval

**Comment by tomayac** [comment]
> @tomayac I always defer to post authors on PRs like this. Does this change look good to you?

Yes, this looks great! Thank you, @tylergaw!

**Review by jpmedley — APPROVED** [review]

**Comment by tylergaw** [comment]
Nice! Thanks everyone.
